template/src/pageComponents/buy-me-coffee/steps/StartTransactionStep/StartTransactionStep.tsx

```diff
@@ -0,0 +1,25 @@
+import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Just to reduce icons library used, can we use https://www.radix-ui.com/icons instead?